packages/custom-functions-metadata/test/cases/requiresstreamparameteraddresses/expected.json

```diff
@@ -0,0 +1,68 @@
+{
+    "allowCustomDataForDataTypeAny": true,
```

Comment:
Let's name this test case requiresparameteraddress-streaming. This is consistent with other test cases in having the tag name first and the variant after the hyphen.